sbnanaobj/StandardRecord/SRCRUMBSResult.h

```diff
@@ -55,7 +55,12 @@ namespace caf {
     ~SRCRUMBSResult() {}
 
     float score;                         //!< CRUMBS result
```

Comment:
Ideally we would change `score` to something more descriptive now that there are other scores, but I guess we want to keep backwards compatibility so expanding the comment to be a bit more descriptive will have to suffice.

Reply:
Yeah this crossed my mind too but a better comment is definitely a good shout!